src/EloquentDataTable.php

```diff
@@ -140,6 +141,19 @@ protected function isMorphRelation($relation)
         return $isMorph;
     }
 
+    protected function getColumnNameByIndex(int $index): string
```

Comment:
I believed this method can be removed since it will be inherited from the QueryDataTable.

Reply:
This is called indirectly from the `CollectionDataTable` as well. I opted to not make it abstract and move the implementation because it will break any userland implementations.